LibreNMS/OS/Zyxelwlc.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ *
+ * @package    LibreNMS
+ * @link       https://www.librenms.org
+ * @copyright  2017 Thomas GAGNIERE
+ * @author     Thomas GAGNIERE <tgagniere@reseau-concept.com>
+ */
+
+namespace LibreNMS\OS;
+
+use LibreNMS\Device\WirelessSensor;
+use LibreNMS\Interfaces\Data\DataStorageInterface;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessApCountDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessClientsDiscovery;
+use LibreNMS\Interfaces\Polling\OSPolling;
+use LibreNMS\OS\Shared\Zyxel;
+use LibreNMS\RRD\RrdDefinition;
+
+class Zyxelwlc extends Zyxel implements OSPolling, WirelessApCountDiscovery, WirelessClientsDiscovery
+{
+    public function pollOS(DataStorageInterface $datastore): void
```

Comment:
You could also do this OID with a "count" sensor in the YAML discovery file. Less code, and no need to create a new graph. That also has the advantage to store the value in database (meaning you can alert on it if necessary). Your current implementation is "Graph only" and more code.
 
But this is of course valid.